e2e-tests/cypress/tests/integration/channels/enterprise/metrics/license_spec.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channels @enterprise @metrics
```

Comment:
`MetricsSettings` is not allowed to be changed in cloud-like test server and so we'll not test this in such environment.

```suggestion
// Group: @channels @enterprise @metrics @not_cloud
```